jet_fuel/test/test_helper.rb

```diff
@@ -0,0 +1,17 @@
+ENV['RAILS_ENV'] ||= 'test'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.